examples/mises_plasticity/mises_plasticity_isotropic_hardening.py

```diff
@@ -1,32 +1,42 @@
+from __future__ import annotations
```

Comment:
Not yet. I currently use ruff (but that should be more or less equivalent to black). I would however ignore the issue in this PR because everything has a consistent format now. We should however decide on some formatting later